src/ansible_navigator/actions/_actions.py

```diff
@@ -52,54 +59,95 @@ def _import_all(package: str) -> None:
     pkg_info = _ACTIONS.setdefault(package, {})
     pkg_info[action] = ActionT(name=action, cls=cls, kegex=re.compile(cls.KEGEX))
     return cls
 
 
 def get(package: str, action: str) -> Callable:
-    """Get a given action"""
+    """Import and return a given action.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :returns: The action's registered class
+    """
     _import(package, action)
     return _ACTIONS[package][action].cls
 
 
 def get_factory(package: str) -> Callable:
-    """Create a get() function for one package"""
+    """Create a ``get`` function for one package.
+
+    :param package: The name of the package
+    :returns: The action's registered class
+    """
     return functools.partial(get, package)
 
 
 def kegex(package: str, action: str) -> Tuple:
-    """Return a tuple of name, kegex for a action"""
+    """Return a tuple of name, class, kegex for a action.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :returns: The name, class and kegex for an action
+    """
     _import(package, action)
     return _ACTIONS[package][action]
 
 
 def kegexes(package: str) -> Generator:
-    """Return a tuple of tuples, name, kegex"""
+    """Return a tuple of tuples, name, kegex for all actions.
+
+    :param package: The name of the package
+    :returns: A generator for all kegexs
+    """
     _import_all(package)
     return (kegex(package, name) for name in names(package))
 
 
 def kegexes_factory(package: str) -> Callable:
-    """Create a kegexs() function for all packages"""
+    """Create a ``kegexs`` function for all packages.
+
+    :param package: The name of the package
+    :returns: A ``kegexes`` method for the package
+    """
     return functools.partial(kegexes, package)
 
 
 def names(package: str) -> List:
-    """List all actions in one package"""
+    """List all actions in one package.
+
+    :param package: The name of the package
+    :returns: All packages
+    """
     _import_all(package)
     return sorted(_ACTIONS[package])
 
 
 def names_factory(package: str) -> Callable:
-    """Create a names() function for one package"""
+    """Create a ``names`` function for one package.
+
+    :param package: The name of the package
+    :returns: a ``names`` method for the package
+    """
     return functools.partial(names, package)
 
 
 def run_interactive(package: str, action: str, *args: Any, **_kwargs: Any) -> Any:
-    """Call the given action's run"""
+    r"""Call the given actions's run method.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :param \*args: The arguments passeed to the action's run method
+    :param \**_kwargs: The keyword arguments passed to the action's run method
+    :returns: The outcome of running the actions' run method
+    """  # noqa: DAR101, DAR102 # *args, **kwargs escaped for sphinx
```

Comment:
The above means that this "noqa" is unnecessary.

Reply:
If I recall it still will be............................................ but we can try

Reply:
yep! nice.